miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/display/MiniAppWebView.kt

```diff
@@ -38,6 +39,11 @@ internal class MiniAppWebView(
         settings.allowUniversalAccessFromFileURLs = true
         settings.domStorageEnabled = true
         settings.databaseEnabled = true
+
+        if (hostAppInfo.isNotEmpty())
+            settings.userAgentString =
+                String.format("%s %s", settings.userAgentString, hostAppInfo)
```

Comment:
Does this UA string should be unique?

Reply:
No, it does not need to be unique. The specs only require the `hostAppInfo` would be appended.